### PR TITLE
[CBRD-21500] Update README to README.md

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CBRD-21500

- [x] change it as README.md
- [x] some graphics may also help
     - [x] Attach CUBRID logo for branding
- [x] Add License section
- [x] Add Contribute section (link it to contributing.md and wiki)
     - a link to cubrid developer guide is added
- [] Add a link to drivers (downloading CUBRID section already covered it)
- [x] HOW TO INSTALL CUBRID and GETTING STARTED sections are duplicated
- [x] Add build from source
   - [x] Add installing build requiements

### CODE REVIEW

**Comment by swi0110** [comment]
How about adding this?
`## Docker container`
`- docker run --name [container name] cubrid:latest`

**Comment by hgryoo** [comment]
@sungc-cubrid Can I have any `.svg` format CUBRID logo? Resizing a `.png` format image causes jaggies.

**Comment by swi0110** [comment]
> @sungc-cubrid Can I have any `.svg` format CUBRID logo? Resizing a `.png` format image causes jaggies.

It might help you. (png convert to svg)
- https://www.pngtosvg.com/
- https://cdkm.com/kr/png-to-svg

**Comment by hgryoo** [comment]
> > @sungc-cubrid Can I have any `.svg` format CUBRID logo? Resizing a `.png` format image causes jaggies.
> 
> It might help you. https://www.pngtosvg.com/

Thank you for the information. But I need an original one. Converting also causes aliasing.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Copyright and license information can be found in the file COPYING.
+
+For more information about CUBRID license policy, visit https://www.cubrid.org/cubrid
+
+## Getting Help
+
+- https://www.reddit.com/r/CUBRID/
+      
+  If You encounter any difficulties with getting started, or just have some questions, or find bugs, or have some suggestions, we kindly ask you to post your thoughts on our subreddit
```
Comment (sungc-cubrid) [inline README.md]: "...any difficulties getting started" will be better; without "with."

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# [<img src="https://ftp.cubrid.org/CUBRID_Docs/Brochure/BI_Images/cubrid-logo-tangram-300.png" width="25" height="25"/>](cubrid_logo.png) CUBRID Database Management System
+
+CUBRID is a DBMS being supported by an active community of open source developers
```
Comment (sungc-cubrid) [inline README.md]: "CUBRID is a DBMS supported by an active..." without the word "being."

File: README.md (hunk 1 above)
Comment (hgryoo) [reply README.md]: Thank you!
```suggestion
  If You encounter any difficulties getting started, or just have some questions, or find bugs, or have some suggestions, we kindly ask you to post your thoughts on our subreddit
```

File: README.md (hunk 2 above)
Comment (hgryoo) [reply README.md]: Thank you
```suggestion
CUBRID is a DBMS supported by an active community of open source developers 
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Copyright and license information can be found in the file COPYING.
+
+For more information about CUBRID license policy, visit https://www.cubrid.org/cubrid
+
+## Getting Help
+
+- https://www.reddit.com/r/CUBRID/
+      
+  If You encounter any difficulties getting started, or just have some questions, or find bugs, or have some suggestions, we kindly ask you to post your thoughts on our subreddit
```
Comment (mhoh3963) [inline README.md]:
```suggestion
  If you encounter any difficulties getting started, or just have some questions, or find bugs, or have some suggestions, we kindly ask you to post your thoughts on our subreddit
```

**Review by mhoh3963 — APPROVED** [review]

**Comment by hgryoo** [comment]
> How about adding this? `## Docker container` `- docker run --name [container name] cubrid:latest`

Sorry for the late answer. I concentrate on how to build from source in this issue. And, IMO, guiding to develop and use CUBRID on docker container env is not yet mature. So I'd like to add about it in the next issue. Thank you for letting me know about the topic.

**Pull request merged**